WoWsShipBuilder.Web/Services/VmCacheService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using WoWsShipBuilder.Web.ViewModels;
+
+namespace WoWsShipBuilder.Web.Services;
+
+public class VmCacheService
```

Comment:
As suggested by you, we should probably rename this to VmCache so it is not confused with an injectable service. Also, some xml doc might not hurt this class